keras_cv/layers/preprocessing3d/global_random_flip_y.py

```diff
@@ -53,26 +53,24 @@ def get_config(self):
     def augment_point_clouds_bounding_boxes(
         self, point_clouds, bounding_boxes, transformation, **kwargs
     ):
-        point_clouds_x = -point_clouds[..., 0:1]
+        point_clouds_y = -point_clouds[..., 1:2]
```

Comment:
Should we rename the layer to FlipX?
And the docstring says "flips on y axis"

Reply:
Done. We could upgrade this function in a future PR to support Flip XY